test/extended/util/framework.go

```diff
@@ -297,6 +297,11 @@ var CheckPodIsRunningFunc = func(pod kapi.Pod) bool {
 	return pod.Status.Phase == kapi.PodRunning
 }
 
+// CheckPodIsSucceededFunc returns true if the pod status is "Succdeded"
+var CheckPodIsSucceededFunc = func(pod kapi.Pod) bool {
```

Comment:
Functions should be named "*Fn" or "fn".  "Func" is reserved for types by
golang convention.

On Tue, Nov 3, 2015 at 8:33 PM, Paul Morie <notifications@github.com> wrote:

> In test/extended/util/framework.go
> <https://github.com/openshift/origin/pull/5632#discussion_r43832610>:
>
> > @@ -297,6 +297,11 @@ var CheckPodIsRunningFunc = func(pod kapi.Pod) bool {
> >  	return pod.Status.Phase == kapi.PodRunning
> >  }
> >
> > +// CheckPodIsSucceededFunc returns true if the pod status is "Succdeded"
> > +var CheckPodIsSucceededFunc = func(pod kapi.Pod) bool {
>
> I see that there are othe rfunctions named *Func but I am pretty against
> this naming convention.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5632/files#r43832610>.
>